### DISCUSSION TITLE
Can't enable Equaliser

### DISCUSSION THREAD

**Discussion by phreatic62** [rern/rAudio#931] [Fixed Issues]
Hi, 
My volume issues seem to have been resolved by making a new build and not restoring a backup, however I now have a problem with the equalizer.
When I select 'Enable' on the Features page, I get a blue message saying 'Equalizer Enable..." that last for a couple of minutes, then it goes away and the Eqlaizer slider goes back to the off position.

re the incompatible settings I shuld be looking for that mean it can;t be enabled?
BT speakers connected and working, in 'Plater' Crossfade,Normanlization and Replay gain are all off. FFmpeg enabled.

Many thanks
Paul

**Comment by Ecelo** [comment]
I have the same problem as you. Just installed yesterday it worked, now the slider is active, the word enable equalizer appears and then it returns to the initial position. I don't know if it's about the bit perfect settings, but I've tried them all and haven't solved it.

**Comment by AndyFx12** [comment]
Same here - and CamillaDSP is not working with HifiBerry DAC+ (errors in config).

**Reply by AndyFx12** [reply]
Update:  Creation of default_config for CamillaDSP is broken. Output format is missing. Changed to S32LE and Camilla works fine. So it's a different issue with equalizer.

**Comment by AndyFx12** [comment]
Fixed with today´s update.

**Reply by phreatic62** [reply]
Yay, so it does. I've remotely updated but can't test with speakers until Tuesday. 😉
